userguide/system.rst

```diff
@@ -583,81 +582,84 @@ The configurable settings are summarized in
    +------------------------------------------+----------------------------------+--------------------------------------------------------------------------------------------------+
-   | Use FQDN for logging                     | checkbox                         | when checked, include the Fully-Qualified Domain Name in logs to precisely identify systems      |
-   |                                          |                                  | with similar hostnames                                                                           |
+   | Use FQDN for logging                     | checkbox                         | Enable to include the Fully-Qualified Domain Name (FQDN) in logs to precisely identify systems   |
+   |                                          |                                  | with similar hostnames.                                                                          |
    |                                          |                                  |                                                                                                  |
    +------------------------------------------+----------------------------------+--------------------------------------------------------------------------------------------------+
-   | Report CPU usage in percentage           | checkbox                         | when checked, CPU usage is reported as percentages in :ref:`Reporting`                           |
+   | Report CPU usage in percentage           | checkbox                         | Enable to display CPU usage as percentages in :ref:`Reporting`.                                  |
    |                                          |                                  |                                                                                                  |
    +------------------------------------------+----------------------------------+--------------------------------------------------------------------------------------------------+
-   | ATA Security User                        | drop-down menu                   | specifies the command account used by legacy :ref:`Self-Encrypting Drives`;                      |
-   |                                          |                                  | choices are *User* or *Master*                                                                   |
+   | ATA Security User                        | drop-down menu                   | User passed to camcontrol security -u for unlocking SEDs. Choices are                            |
+   |                                          |                                  | *User* or *Master*.                                                                              |
    |                                          |                                  |                                                                                                  |
```

Comment:
s/camcontrol security -u/:samp:`camcontrol security -u`/

Reply:
github is being helpful here and I don't see how to turn it off. That is :samp:backtick camcontrol security -u backtick